Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -151,6 +148,16 @@ public enum LayoutType
         [SerializeField]
         private float m_MinDistance = 0.25f;
 
+        /// <summary>
+        /// Inactivity time that makes the keyboard disappear automatically.
+        /// </summary>
+        public float CloseOnInactivityTime = 15;
```

Comment:
Should this feature be locked behind a `bool` setting, in case I don't want the keyboard to close automatically?